### PR TITLE
minor typos in thick-skin

### PR DESCRIPTION
**Pull request by kenegozi**
cray -> crazy and a missing space

### CODE REVIEW

File: _posts/archived/2012-12-17-you-do-not-need-a-thick-skin.aspx.markdown [hunk 1]
```diff
@@ -74,10 +74,10 @@ Angry. Perhaps I said a few words I’m not proud of about how I might
 inflict bodily harm on the author in anatomically impossible ways should
 we ever meet in a dark alley.
 
-I certainly didn’t mean those words (*except in the case of*[*bundled
+I certainly didn’t mean those words (*except in the case of* [*bundled
 software written by hardware
 companies*](http://www.codinghorror.com/blog/2009/07/nobody-hates-software-more-than-software-developers.html "Nobody hates software more than software developers")*.
-That shit makes me cray!*). I was simply lashing out due to my
```
Comment (Haacked) [inline _posts/archived/2012-12-17-you-do-not-need-a-thick-skin.aspx.markdown]: https://www.urbandictionary.com/define.php?term=Cray

**Comment by kenegozi** [comment]
oh man how I miss using github for code-reviews! edit right from the PR and the commit joins the changelist automagically! This shit is cray!

**Comment by Haacked** [comment]
> oh man how I miss using github for code-reviews! edit right from the PR and the commit joins the changelist automagically! This shit is cray!

Ha ha! What are you using these days?

**Comment by Haacked** [comment]
Thanks!

**Pull request merged**